.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: c
+sudo: required
+dist: trusty
+addons:
+  apt:
+    packages:
+    - libsdl1.2-dev
+    - libjson-glib-dev
+    - libglib2.0-dev
+    - libgif-dev
+    - libpango1.0-dev
+    - librsvg2-dev
+    - libsdl1.2-dev
+    - openexr
+script:
+- make release-and-upload VERSION=`git describe --tags` TARGET=travis-${TRAVIS_OS_NAME}
```

Comment:
This should call a `check` or `release-check` target. Upload should be separate.